cmd/humanInsertion/humanInsertion.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/maf"
+	"log"
+)
+
+func humanInsertion(mafFile string, outBed_ins string, outBed_del string, species_ins string, species_del string) {
+	//initialize variables
+	mafRecords := maf.Read(mafFile) //Read entire mafFile. mafRecords has type Maf
+	var bedList_ins []*bed.Bed      //initialize 2 bed files
+	var bedList_del []*bed.Bed      //1 bed file for ins, 1 bed file for del
+
+	//go through each line
+	for i, _ := range mafRecords { //each i is a block
+		for k, _ := range mafRecords[i].Species { //each k is a line
```

Comment:
We are looping k over all species in the alignment here, but later on there is a built in assumption that k=0 corresponds to  species_ins and k=1 corresponds to species_del.  Maybe it makes sense to start the k loop at 1 instead of 0 and then change the k-1 indices to 0?
The loop would be something like for k=1; k<len(mafRecords[i].Species); k++ {